lib/bonny/crd.ex

```diff
@@ -100,16 +99,21 @@ defmodule Bonny.CRD do
     version: v1
   ```
   """
-  @spec to_manifest(Bonny.CRD.t()) :: map
-  def to_manifest(%CRD{} = crd) do
+  @spec to_manifest(Bonny.CRD.t(), String.t()) :: map
+  def to_manifest(%CRD{} = crd, api_version) do
```

Comment:
We may need to do a default value on `api_version` here since its a public method.